src/commands/Notifications/youtube-subscription.ts

```diff
@@ -0,0 +1,162 @@
+import { envParseBoolean } from '#lib/env';
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import type { GuildMessage } from '#lib/types';
+import { PermissionLevels } from '#lib/types/Enums';
+import { hyperlink } from '@discordjs/builders';
+import { ApplyOptions, RequiresClientPermissions } from '@sapphire/decorators';
+import { Args, CommandOptionsRunTypeEnum, Resolvers } from '@sapphire/framework';
+import { send } from '@sapphire/plugin-editable-commands';
+import { PermissionFlagsBits } from 'discord-api-types/v9';
+import { MessageEmbed } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	enabled: envParseBoolean('GRPC_NOTIFICATIONS_ENABLED'),
+	aliases: ['y-subscription', 'y-sub'],
```

Comment:
```suggestion
	aliases: ['y-subscription', 'y-sub', 'yt-sub'],
```

Might as well be complete then